api/libs/cost-surfaces/src/cost-surface-pu-data.geo.entity.ts

```diff
@@ -20,7 +20,7 @@ export class CostSurfacePuDataEntity {
     (projectsPu: ProjectsPuEntity) => projectsPu.id,
   )
   @JoinColumn({ name: 'puid' })
-  projectsPu!: ProjectsPuEntity;
+  projectsPu?: ProjectsPuEntity;
 
   @Column({ type: 'uuid', name: 'puid' })
   /**
```

Comment:
I totally agree with that, this was done as it is because it was specified in the relevant task for it, (in which I suggested that the naming might be confusing) but my understanding of the domain was poor enough to not to raise my voice at the time.

In order to modify this, we will need another migration, which should be added wherever see best, taking in account: https://github.com/Vizzuality/marxan-cloud/pull/1502#discussion_r1335352553

Reply:
apologies for adding to the confusion 😬